configuration/restdocs.md

```diff
@@ -7,14 +7,38 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+Through the openHAB [REST API](https://en.wikipedia.org/wiki/REST_API) most aspects of the openHAB system can be accessed by other programs.  
+This includes for example the access to all data related to Items, Things and Bindings as well as the capabilities to invoke  actions that change the state of Items. Interactions with the REST API are based on a simple http protocoll.
+The REST API provides an interface that can only be accessed programmatically.
+The information available from the API includes Items and their state, installed bindings, sitemap information, and much more. 
+The information exchange takes place through HTTP and the API provides a defined protocoll for information exchange among different computer systems. 
+However, from a technical perspective it is very easy to use. 
+Be aware that the REST API is not automatically installed in openHAB. 
```

Comment:
I think in my version I briefly mention how to install it.  This is important as someone may be a new user and have come directly here to start working.  So a brief mention of where it is located would be good.  I think the wording I used after @ThomDietrich 's comment was:

Be aware that the REST API is not automatically installed in openHAB. After the REST API documentation add-on is installed, it is accessible through the openHAB dashboard.